docs/Local-Development.md

```diff
@@ -0,0 +1,162 @@
+# Local development
+
+1. [First steps | Local installation](#first-steps)
+1. [Setup testing env](#setup-testing-env)
+1. [Test commands locally](#test-commands-locally)
+1. [Cleanup](#cleanup)
+
+## First Steps
+
+**Get an OpsLevel API token for a dev environment.** Do not use your prod environment since this involves mutating lots of services in OpsLevel.
+
+Install these:
+
+1. [minikube](https://minikube.sigs.k8s.io/docs/start/)  - this is a suggestion, you can also use **your own dev cluster if you have one.**
+1. [task](https://taskfile.dev/) - this will install dependencies for you.
+1. [OpsLevel CLI](https://github.com/OpsLevel/cli) - makes things way easier.
+
+```
+brew install minikube
```

Comment:
😢 - do we really use minikube - https://kind.sigs.k8s.io/ is way better and less of a resource hog

Reply:
(: minikube is mentioned in a few k8s docs and I thought it was really easy to install even if it is a resource hog, I'll mention that is also an option along with it.